aws-servicebroker/Makefile

```diff
@@ -1,54 +1,104 @@
-DOCKER_REPOSITORY ?= splatform
-DOCKER_TAG ?= latest
+DOCKER_REPOSITORY ?= awsservicebroker
+DOCKER_TAG ?= beta
 NAMESPACE ?= aws
 INSTANCE_NAME ?= mysql-inst
+BROKER_VERSION ?= 1.0.0-beta.3
 BROKER_USERNAME ?= username
 BROKER_PASSWORD ?= password
-REGISTRY_HOSTNAME ?= ${empty}
-REGISTRY_USERNAME ?= ${empty}
-REGISTRY_PASSWORD ?= ${empty}
-IAM_ROLE ?= $(shell aws iam list-roles --output=json | jq -r '.Roles[] | select(.RoleName | test("ServiceBroker"; "i")) | .Arn' | head -n1)
+CF_NAMESPACE ?= cf
 
-# Use `make COMMAND=install` to see the generated kube resources (for debugging)
-COMMAND ?= upgrade --install --reuse-values --recreate-pods --force
+REGION ?= us-west-2
+AWS_VPC ?= $(shell aws ec2 describe-vpcs --region "${REGION}" --output json | \
+    jq -r '                                                                   \
+        .Vpcs[] |                                                             \
+        select(                                                               \
+            .Tags // [] |                                                     \
+             .[] |                                                            \
+             select(.Key == "Name") |                                         \
+             .Value |                                                         \
+             contains("awssb")                                                \
```

Comment:
Is this condition something we should note in the README, in the step talking about the need to create a VPC ?

Reply:
Ah, yes it is.

Reply:
Looks good.